plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelConfiguratorUI.java

```diff
@@ -758,7 +758,7 @@ public SSHHostConfiguration saveSettings() throws DBException {
                 case PUBLIC_KEY -> {
                     final String privateKey = privateKeyText.getText().trim();
                     if (DBWorkbench.isDistributed()) {
-                        yield new SSHAuthConfiguration.KeyData(privateKey, password, savePassword);
+                        yield new SSHAuthConfiguration.KeyData(SSHUtils.getKeyData(privateKey), password, savePassword);
```

Comment:
I think it's better to do this inside `SSHUtils#saveHostConfiguration`

Reply:
we don't use `SSHUtils#saveHostConfiguration` in CloudBeaver

Reply:
But we do. This particular class (`SSHTunnelConfiguratorUI`) is only used in the desktop product, hence it makes sense to perform sanitization where the data is actually serialized/deserialized.

Reply:
Actually, we don't need to trim key data here - it will be trimmed on handler init.